src/planning/src/planning/track.cpp

```diff
@@ -15,7 +15,59 @@ Track::~Track() {
   for (int i = 0; i < rSize; i++) delete rightCones[i];
 }
 
+void Track::logCones(bool side) {
+  if (side) {
+    std::cout << "LEFT CONES: ";
```

Comment:
Any reason to use std::cout instead of logging?

Reply:
This function was only used for visualizing the data with Desmos and it formats the cones' coordinates accordingly. To clarify, I left it as a method of the Track class just because it may be useful for debugging, even though it is currently only used for testing.

Reply:
Logging doesn't work because either it prints each cone in one line or it truncates most of the cones because there is a maximum number of characters in each log.